app/main/views.py

```diff
@@ -11,6 +12,9 @@
 from ..single_resource.forms import SingleResourceForm
 
 from datetime import datetime
+ACCOUNT_SID = "AC66b23a9fb924a84547e9c95e56436895" 
```

Comment:
can you put these in the `.env` file? and then reference these by 
`os.environ.get('ACCOUNT_SID')` etc?
also import os